pkg/lib/form-layout.less

```diff
@@ -56,6 +56,7 @@
   // Compute widget height based on (font-size * line height) + extra
   @widget-height: @line-height-computed + (@padding-base-vertical * 2) + (@border-width * 2);
 
+  align-self: start; // Don't vertically fill content by default
```

Comment:
The vertical alignment not working between the two grids was the thing that threw me off the most (outside of getting my system properly building Cockpit again after being away for a bit).

It turns out, we could either align the content in the parent grid to the start, or tell the grid to align itself to the start. I chose the latter, so we could include a grid inside a grid (even a `ct-form-layout` inside another) or inside a flex... and things should "just work" as expected.